README.md

```diff
@@ -31,33 +50,47 @@ _Note_: These tools are intended for testing and as reference implementation of
 
 ## Getting Started
 
-These instructions will get you a copy of the project up and running on your local machine for development and testing purposes. See Docker for instructions on how to test against network equipment.
+These instructions will get you a copy of the project up and running on your local machine.
 
 ### Prerequisites
 
 Install __go__ in your system https://golang.org/doc/install. Requires golang1.7+.
 
-### Clone
+### Download sources
 
-Clone the project to your __go__ source folder:
 ```
-mkdir -p $GOPATH/src/github.com/google/
-cd $GOPATH/src/github.com/google/
-git clone https://github.com/google/gnxi.git
+go get -v github.com/google/gnxi/...
 ```
 
-### Running
+### Building and installing binaries
+
+```
+cd $GOPATH
+mkdir bin
+go install github.com/google/gnxi/...
+ls -la $GOPATH\bin
```

Comment:
Should we recommend the user add $GOPATH/bin to their $PATH? So that they can use the binaries anywhere